src/openforms/prefill/base.py

```diff
@@ -50,6 +51,26 @@ def get_prefill_values(
         """
         raise NotImplementedError("You must implement the 'get_prefill_values' method.")
 
+    @classmethod
+    def get_prefill_values_from_mappings(
+        cls, submission: Submission, form_variable: FormVariable
+    ) -> dict[str, str]:
+        """
+        Given the saved form variable, which contains the prefill_options, look up the appropriate
+        values and return them.
+
+        :param submission: an active :class:`Submission` instance, which can supply
+          the required initial data reference to fetch the correct prefill values.
+        :param form_variable: The form variable for which we want to retrieve the data. Its
+        atribute prefill_options contains all the mappings that are needed for retrieving
+        and returning the values.
+        :return: a key-value dictionary, where the key is the mapped property and
+          the value is the prefill value to use for that property.
```

Comment:
```suggestion
        :return: a mapping where the keys are form variable keys, and the values are the 
          initial/default values to assign to the matching form variable. The variable keys
          can point to both component and user defined variables.
```
I think this is how it's meant?